packages/next/src/lib/metadata/resolve-metadata.test.ts

```diff
@@ -461,6 +461,62 @@ describe('accumulateMetadata', () => {
         },
       })
     })
+
+    it('should inherit metadata title description into openGraph or twitter if they are configured', async () => {
+      const metadataItems1: MetadataItems = [
+        [
+          {
+            title: 'My title',
+            description: 'My description',
+            openGraph: {
+              images: 'https://test.com/og.png',
+            },
+          },
+          null,
+        ],
+      ]
+      const metadata1 = await accumulateMetadata(metadataItems1)
+      expect(metadata1).toMatchObject({
+        openGraph: {
+          title: {
+            absolute: 'My title',
+            template: null,
+          },
+          description: 'My description',
+        },
+        twitter: {
+          title: {
+            absolute: 'My title',
+            template: null,
+          },
+          description: 'My description',
+        },
+      })
+
+      const metadataItems2: MetadataItems = [
+        [
+          {
+            title: 'My title',
+            description: 'My description',
+            twitter: {
+              images: 'https://test.com/twitter.png',
+            },
```

Comment:
it might be worth adding a test that if you explicitly set `twitter.title` or `twitter.description`, that `metadata.title` / `metadata.description` don't override it (we might already have a test for this, in which case ignore me :D)